### ISSUE TITLE
Add a hub link to Atom and RSS feeds

### ISSUE THREAD

**Issue by lionirdeadman** [getzola/zola#1213]
This is a fairly minor change to the built-in but I'd like to have something these lines in the atom and rss built-ins
```xml
    <link href="{{ config.extra.publisher_url }}" rel="hub" />
```
Why should that be there?

This link allows for WebSub hubs to be specified in the RSS/Atom which is neccessary as per WebSub specs.

Why should zola support WebSub?

WebSub is quite useful to Feed readers since it means they no longer have to read the feed file every x amount of time and can instead just wait for the hub to tell them to do so which makes it much more efficient.

(I could've just made a PR but it said that I should make an issue hence here we are, I hope I explained this well enough)

**Comment by Keats** [comment]
Isn't that part of the template though? What do we need to do in Zola itself?

**Comment by lionirdeadman** [comment]
Yeah, it would part of the template, it's just missing from the built-in template. I'd just be modifying it. Do you think it would be out-of-scope?

**Comment by Keats** [comment]
Which built-in template?

**Comment by lionirdeadman** [comment]
[atom.xml](https://github.com/getzola/zola/blob/master/components/templates/src/builtins/atom.xml) and [rss.xml](https://github.com/getzola/zola/blob/master/components/templates/src/builtins/rss.xml)

They have rel=self but don't offer the option for rel=hub (which is needed for WebSub)

**Comment by Keats** [comment]
It would be ok to add.

**Comment by teotwaki** [comment]
@lionirdeadman is this still relevant?

**Comment by lionirdeadman** [comment]
I no longer use Zola. It's fairly harmless to my knowledge to overwrite the built-in templates for atom.xml and rss.xml though so for those interested in WebSub, I think they can figure it out. If you want to provide support to specify the `rel=hub` in these files, then this issue is still valid to track this feature.

All this to say, up to you.
